### PR TITLE
env: enable automated package deployment

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- This PR is a setup for the automated package deployment.
- Requires a commit having a message as the new version.
- Created a small script following [Editor's one](https://github.com/nhn/tui.editor/blob/master/scripts/publish-cdn.js)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by dongsik-yoo — COMMENTED** [review]
수고 많으셨습니다. 점점 더 좋아지고 편해지네요.

File: .github/actions/get-target-version/action.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: 'Get target version'
+description: "Find the target release version"
+outputs:
+  target-version:
+    value: ${{ steps.target-version-getter.outputs.target-version }}
+runs:
+  using: "composite"
```
Comment (js87zz) [inline .github/actions/get-target-version/action.yml:7]: 재사용하는 방법을 못 찾았었는데 이거였군요..ㅋㅋ공식 문서를 더 자세히 찾아볼 걸 그랬군요.

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다!

File: .github/actions/get-target-version/action.yml (hunk 1 above)
Comment (adhrinae) [reply .github/actions/get-target-version/action.yml:7]: 이런 방식으로 사용할 수 있게 된게 비교적 최근에 추가된 것 같더라고요.

https://github.blog/changelog/2021-08-25-github-actions-reduce-duplication-with-action-composition/

Comment (js87zz) [reply .github/actions/get-target-version/action.yml:7]: 컥..그래서 못 찾았었나 보네요.. 훨씬 낫네요ㅋㅋㅋ에디터도 나중에 적용해야겠군요

**Pull request merged**